starport/templates/typed/typed.go

```diff
@@ -1,33 +1,17 @@
 package typed
 
 import (
-	"embed"
 	"strings"
 
 	"github.com/gobuffalo/genny"
 	"github.com/gobuffalo/packd"
 	"github.com/gobuffalo/plush"
 	"github.com/gobuffalo/plushgen"
 	"github.com/tendermint/starport/starport/pkg/plushhelpers"
-	"github.com/tendermint/starport/starport/pkg/xgenny"
 	"github.com/tendermint/starport/starport/pkg/xstrings"
 	"github.com/tendermint/starport/starport/templates/testutil"
 )
 
-var (
-	//go:embed stargate/component/* stargate/component/**/*
-	fsStargateComponent embed.FS
-
-	//go:embed stargate/messages/* stargate/messages/**/*
-	fsStargateMessages embed.FS
-
-	// stargateComponentTemplate is the template for a Stargate module type component
-	stargateComponentTemplate = xgenny.NewEmbedWalker(fsStargateComponent, "stargate/component/")
-
-	// stargateMessagesTemplate is the template for a Stargate module type interaction messages
-	stargateMessagesTemplate = xgenny.NewEmbedWalker(fsStargateMessages, "stargate/messages/")
-)
-
 func Box(box packd.Walker, opts *Options, g *genny.Generator) error {
```

Comment:
may we move the "strconv" context method below to `plushhelpers` package

Reply:
`strconv` returns true if `strconv` package must be imported (when the type has at least `string` field). So it seems very specific the type templates and placeholders. I would personally keep it here since it's not enough generic